.cloudbuild/hashicorp.repo

```diff
@@ -1,13 +1,13 @@
 [hashicorp]
 name=Hashicorp Stable - $basearch
-baseurl=https://rpm.releases.hashicorp.com/RHEL/7/$basearch/stable
+baseurl=https://rpm.releases.hashicorp.com/RHEL/8/$basearch/stable
```

Comment:
Is there a reason we choose 8 over 9?

Reply:
Was being conservative with the initial update to avoid unintended behavior since we are still on an old distribution of oracle linux for the graalvm image. However, I think rhel 9 works.